sklearn/metrics/classification.py

```diff
@@ -221,8 +221,10 @@ def confusion_matrix(y_true, y_pred, labels=None, sample_weight=None):
 
     References
     ----------
-    .. [1] `Wikipedia entry for the Confusion matrix
+    .. [1] `Wikipedia entry for the Confusion matrix*
            <https://en.wikipedia.org/wiki/Confusion_matrix>`_
+           *NOTE: Different conventions exist for confusion matrix axes.
+           ``confusion_matrix`` gives true class along axis=1.
```

Comment:
I think "along axis=1" must be ambiguous. Because as far as I can tell, each true class corresponds to a row (i.e. axis=0). Please word this differently.

Reply:
You're right-- my mistake. Since the docstring does explain the output of ```confusion_matrix``` , maybe I only make the Wikipedia note in the docstring to avoid redundancy, and explain the axes convention in the user guide? What do you think?

```rst
Confusion matrix
----------------

The :func:`confusion_matrix` function evaluates
classification accuracy by computing the confusion matrix
with each row corresponding to the true class
<https://en.wikipedia.org/wiki/Confusion_matrix>`_.
(Wikipedia may use different convention for axes.)